src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -371,7 +371,7 @@ void BaseQNAcceleration::performAcceleration(
     /**
      * apply quasiNewton update
      */
-    _values = _oldValues + xUpdate + _residuals; // = x^k + delta_x + r^k - q^k
```

Comment:
These lines are mathematically and programming wise equivalent as far I can tell. I checked the pullrequest #703 where the manifold-mapping was removed and this update formula seems to come from there. As far as I can understand manifold-mapping feature was some sort of "multigrid method" with a fine and a coarse model. The q^k referenced in the comment was used to get the correct optimization target for the coarse model. 

I would be in favor for chainging this, since "_values += xUpdate;" is a lot closer to what I am doing in #1834 and would allow a easier comparison between rQNWR and preCICE current QN acceleration.